src/components/FolderInfo/FolderInfo.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled from "styled-components";
+
+export interface folderInfoProps {
+  ownerName: string;
+  folderName: string;
+  profileImgSrc?: string;
+}
+
+const DEFAULT_PROFILE_IMG_SRC = "assets/images/avatar.png";
+
+const FolderInfo: React.FC<folderInfoProps> = ({
```

Comment:
```suggestion
const FolderInfo = ({
```
중요한건 아니지만 React.FC는 사용하지 않아도 돼서 안쓰는 추세입니다.